packages/connect-web/src/popup/index.ts

```diff
@@ -110,19 +110,17 @@ export class PopupManager extends EventEmitter {
             if (this.settings.env === 'webextension') {
                 chrome.tabs.get(this.popupWindow.id, tab => {
                     if (!tab) {
-                        this.close();
-                        this.emit(POPUP.CLOSED);
+                        this.clear();
                     }
                 });
             } else if (this.popupWindow.closed) {
-                this.close();
-                this.emit(POPUP.CLOSED);
```

Comment:
This PR removes 
```
this.emit(POPUP.CLOSED);
```
from some places but keeps it on another places. What is the reason? 

Imho this event should be emitted at some point of time. I am not sure whether it should be part of clear or close method. But it is currently used in core to trigger `interruptionFromUser` so I guess it should stay?

Reply:
Yes, I agree `this.emit(POPUP.CLOSED);` should be there.
I updated and rebased.